### PR TITLE
[Blog] Add plugin list + news

### PR DESCRIPTION
**Pull request by matmair**
This PR adds a blog entry for the plugin list and a news entry for the month of January.

### CODE REVIEW

File: _posts/2023-01-30-plugin-list.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+---
+author: matmair
+title: Starting the Plugin List
+---
+
+Recently a new feature was launched: a central plugin list.
+
+### TLDR
+
+The Plugin List is managed by the [InvenTree org](https://github.com/inventree) and [open for submissions](https://github.com/inventree/inventree-website#adding-a-plugin-to-the-list). It is a simple list for now. Beeing on the list is now an endorsement by the core team.
```
Comment (SchrodingersGat) [inline _posts/2023-01-30-plugin-list.md:10]: "Being" not "beeing" :)

**Comment by matmair** [comment]
@SchrodingersGat would you mind cross-reading this and merging it if it is fine?

**Comment by SchrodingersGat** [comment]
@matmair this link appears to be broken (in the netlify preview)

image

**Comment by SchrodingersGat** [comment]
Actually, all these links are broken. Is this just because it is a preview?

image

**Comment by matmair** [comment]
oops , thank you for checking. The preview should be exactly the same as the final deployment, just hosted on another domain.

**Comment by SchrodingersGat** [comment]
Nice, thanks @matmair

**Pull request merged**